sdks/java/io/mongodb/src/main/java/org/apache/beam/sdk/io/mongodb/MongoDbIO.java

```diff
@@ -166,7 +165,7 @@ private MongoDbIO() {}
 
     abstract SerializableFunction<MongoCollection<Document>, MongoCursor<Document>> queryFn();
 
-    abstract Builder builder();
+    abstract Builder toBuilder();
```

Comment:
This rename is slightly unreated to this PR but an improvement to be consistent with [AutoValue conventions](https://github.com/google/auto/blob/master/value/userguide/builders-howto.md#to_builder).